NebulaPatcher/Patches/Dynamic/PlanetModelingManager_Patch.cs

```diff
@@ -66,6 +67,12 @@ public static bool RequestLoadStar(StarData star)
 
             InternalLoadPlanetsRequestGenerator(star.planets);
 
+            // Request initial dysonSphere data
+            if (GameMain.data.dysonSpheres[star.index] == null)
+            {
+                Log.Info($"Requesting DysonSphere for system {star.displayName} (Index: {star.index})");
+                LocalPlayer.SendPacket(new DysonSphereLoadRequest(star.index));
+            }
```

Comment:
So, if I understand correctly, we only request it once. We will add incremental updates later ?